tests/support/fixtures.ts

```diff
@@ -129,18 +131,38 @@ export const test = base.extend<{
     await logFile.close();
   },
 
+  peerManager: async ({ stateDir, outputLog }, use) => {
+    const peerManager = await createPeerManager({
+      dataDir: Path.resolve(Path.join(stateDir, "peers")),
+      outputLog,
+    });
+    await use(peerManager);
+  },
+
   // eslint-disable-next-line no-empty-pattern
   stateDir: async ({}, use, testInfo) => {
-    const stateDir = testInfo.outputDir;
+    // Ok, we're moving the state dir to /tmp to avoid hitting the SUN_LEN limit, due to socket files.
+    // We still create a symlink in the testInfo.outputDir, so that we can easily find the state dir.
```

Comment:
Ideally we would make the `control.sock` path configurable on `radicle-node`. And then only create a temporary OS tmp dir for that, instead all the spec state. This way once the test has run, the OS tmp dir with the socket would be cleaned up automatically. Right now all the state still remains in the system `/tmp` dir until the machine gets rebooted.

Here's some inspiration on how we did it in Upstream: https://github.com/radicle-dev/radicle-upstream/blob/main/test/support/peerManager.ts#L305.

Reply:
hmm I think one thing that we can propose is to have a `RAD_SOCKET_DIR` env variable or something similar that would only work with debug binaries. But will have to look into that first in heartwood

Reply:
bbcf9ac